src/api/index.js

```diff
@@ -57,10 +57,16 @@ export function getKubeAPI(
   ].join('');
 }
 
-export function getTektonAPI(type, { name = '', namespace } = {}, queryParams) {
+export function getResourcesAPI(
+  group,
```

Comment:
I think it's time to move all of these into the object with `name` and `namespace`

```js
export function getResourcesAPI(
  {
    group,
    name,
    namespace,
    type,
    version
  },
  queryParams
) {
```

no need for the default empty object since at least one of those values will always be required, also makes the consuming code easier to read (named params)

also means `getCustomResource` and `getCustomResources` can just pass them through, no need for them to be aware of the contents

```js
export function getCustomResources(...args) {
  const uri = getResourcesAPI(...args);
  return get(uri).then(checkData);
}
```